src/desktop/components/main_layout/templates/experimental_app_shell.jade

```diff
@@ -0,0 +1,29 @@
+extends blank
```

Comment:
We've got a zillion "new" templates in this folder, at some point we need to clean em up...